src/main/java/com/cloudbees/jenkins/plugins/bitbucket/impl/endpoint/AbstractBitbucketEndpoint.java

```diff
@@ -182,41 +194,6 @@ public String getEndpointJenkinsRootUrl() {
         }
     }
 
-    /**
-     * Look up in the current endpoint configurations if one exists for the
-     * serverUrl, and return its normalized endpointJenkinsRootUrl value,
-     * or the normalized global default Jenkins Root URL if nothing was found
-     * or if the setting is an empty string; empty string if there was an error
-     * finding the global default Jenkins Root URL value (e.g. core not started).
-     * This is the routine intended for external consumption when one needs a
-     * Jenkins Root URL to use for webhook configuration.
-     *
-     * @param serverURL Bitbucket Server URL for the endpoint config
-     *
-     * @return the normalized custom or default Jenkins Root URL value
-     */
-    @NonNull
-    public static String getEndpointJenkinsRootUrl(String serverURL) {
```

Comment:
@KalleOlaviNiemitalo this was the old logic. It's the same of actual. If Jenkins endpoint URL has not been customised for this endpoint than it use the Jenkins default set in Manage Jenkins -> Jenkins Location -> Jenkins URL